packages/calypso-products/src/constants/akismet.ts

```diff
@@ -91,6 +91,6 @@ export const AKISMET_UPGADES_PRODUCTS_MAP = {
 	[ PRODUCT_AKISMET_PLUS_20K_YEARLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_PLUS_30K_YEARLY }`,
 	[ PRODUCT_AKISMET_PLUS_30K_MONTHLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_PLUS_40K_MONTHLY }`,
 	[ PRODUCT_AKISMET_PLUS_30K_YEARLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_PLUS_40K_YEARLY }`,
-	[ PRODUCT_AKISMET_PLUS_40K_MONTHLY ]: 'https://akismet.com/enterprise',
-	[ PRODUCT_AKISMET_PLUS_40K_YEARLY ]: 'https://akismet.com/enterprise',
+	[ PRODUCT_AKISMET_PLUS_40K_MONTHLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_ENTERPRISE_MONTHLY }`,
+	[ PRODUCT_AKISMET_PLUS_40K_YEARLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_ENTERPRISE_YEARLY }`,
```

Comment:
I believe we'll have to then map [ PRODUCT_AKISMET_ENTERPRISE_YEARLY ] to `https://akismet.com/enterprise` because there are 3 more levels of enterprise that won't be offered through checkout

Reply:
Thanks for making a good point! I wasn't aware of this strategy (and /enterprise was resulting in 404 but I guess it will change in the new version of the site).

Reply:
Yep, it will work when the site is live 😄  ( In theory 😅  )